RIOT-OS/RIOT
# ble/nimble: bump version to 9d4bda2

### Contribution description
The current NimBLE version used has an issue when receiving large data packets via L2CAP COC. This was fixed upstream (https://github.com/apache/mynewt-nimble/pull/378). So this PR updates NimBLE to include those fixe(es).

### Testing procedure
Once #11235 is merged, use that test application to verify that sending/receiving large data packets at a high rate is working properly.

### Issues/PRs references
fixes https://github.com/apache/mynewt-nimble/issues/373
depends on #11235 (for proper testing)
